docs/sources/explore/logs-integration.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Logs in Explore"
+description = "Logs in Explore"
+keywords = ["explore", "logs",]
+aliases = ["/docs/grafana/latest/features/explore/"]
```

Comment:
Only one file should have the alias for this page: the _index page. Otherwise, Hugo will get confused about where to redirect people.

Reply:
Removed aliases from subtopics.